### ISSUE TITLE
IOS error: pod install failed with 'use_frameworks!'  

### ISSUE THREAD

**Issue by sosospicy** [Sh1d0w/multi_image_picker#30]
**Describe the bug**
I'm NOT sure it is about this plugin, or a flutter problem [#14161](https://github.com/flutter/flutter/issues/14161) , or other package problem.
The example of this plugin works great on IOS.
Everything works greate on Android.
But on IOS, when work along with another package( in my case with [tflite](https://github.com/shaqian/flutter_tflite) ), there will be a build error which seems related to this config `'use_frameworks!'` .

With `'use_frameworks!'` in podfile, run `pod install` or `flutter run`, got this error:
`
The 'Pods-Runner' target has transitive dependencies that include static binaries: (test_app/ios/Pods/TensorFlowLite/Frameworks/tensorflow_lite.framework)
`
With `'use_frameworks!'` commented out in podfile, run `flutter run`, got this error:
`
/mypath/development/flutter/.pub-cache/hosted/pub.flutter-io.cn/multi_image_picker-2.2.54/ios/Classes/MultiImagePickerPlugin.m:2:9: fatal error: 'multi_image_picker/multi_image_picker-Swift.h'
    file not found
    #import <multi_image_picker/multi_image_picker-Swift.h>
`

**To Reproduce**
1. create a flutter application with  `flutter create -i swift  test_app`
2. edit pubspec.yaml
`
  multi_image_picker: ^2.2.54
  image_picker:
  tflite:
`
3. edit ios/Podfile
`
  platform :ios, '10.0' <--- edit here
  target 'Runner' do
    use_frameworks! <--- edit here
`
4. copy multi_image_picker example/lib/* to test_app/lib/

```
<!-- Insert example code here, that will help to reproduce the bug -->
```

**Expected behavior**

**Screenshots**

**Flutter (please complete the following information):**
- Which version of `multi_image_picker` are you using: [e.g. 2.0.04]
   2.2.54
```
 <!-- run `flutter doctor` and paste the output here: -->
```
  Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, v0.11.3, on Mac OS X 10.14.1 18B75, locale en-CN)
[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
[✓] Android Studio (version 3.2)
[✓] Connected device (1 available)
**iOS (please complete the following information):**
 - Device: [e.g. iPhone6] .   iPhone6 plus
 - OS: [e.g. iOS8.1] . IOS12.0
 - Swift Version: [e.g. 4.0]  4.2

**Android (please complete the following information):**
 - Device: [e.g. Nexus 5]
 - OS: [e.g. Lollipop 5.1]

**Additional context**

**Comment by Sh1d0w** [comment]
Hi, and thanks for opening the issue.

I was able to reproduce the problem, but it does not seem to be caused by my plugin (as it works just fine when the other plugin is not present).

When you create your project with `flutter create -i swift test_app` it will automatically add `use_frameworks!` to the pod file, which is needed when developing packages with Swift.

It seems like the problem is in the other plugin not working correctly when `use_frameworks!` is there, so you can open an issue on the other plugin's repo and also track the status of https://github.com/flutter/flutter/issues/14161 to see when it will be fixed as well.

**Comment by sosospicy** [comment]
OK. And thank your for helping me make it clear.

**Comment by Sh1d0w** [comment]
@sosospicy Actually I think I've found a solution, just published `2.2.55`, can you try it and let me know if it works for you? You can now remove `use_frameworks!` it should work without that.
